app/views/devise/registrations/new.html.erb

```diff
@@ -14,11 +14,8 @@
   </div>
 
   <div class="field">
-    <%= f.fields_for :department do |d| %>
-      <%= d.label :department %>
-      <%= d.select :name,
-          Department::DEPARTMENT, class: "department dropdown"%>
-    <% end %>
+      <%= f.label :department_id %>
+      <%= collection_select(:department_id, :department_id, Department.all, :id, :name) %>
```

Comment:
Yeah, the options we need to use are `object, method, collection, value_method, text_method` based off those options on form submission we are returned with `"object"=>{"method"=>":id"}`... We used `collection_select` because we needed to pull in the options for the select from the departments table in the database. Maybe we should use a different method here instead that could do the same thing that wouldn't submit a hash? Or is there a way to add only the value from the hash to the column in that table of the db? Also, we tried removing and changing the options and its mandatory to have those 5.